config/fcos/v1_6_exp/validate.go

```diff
@@ -15,13 +15,26 @@
 package v1_6_exp
 
 import (
+	"regexp"
+
 	"github.com/coreos/butane/config/common"
 	"github.com/coreos/ignition/v2/config/util"
 
 	"github.com/coreos/vcontext/path"
 	"github.com/coreos/vcontext/report"
 )
 
+var regex = regexp.MustCompile(`^/(etc|var)/.*`)
```

Comment:
Nit: the trailing `.*` is redundant.

Note https://github.com/coreos/butane/issues/240#issuecomment-1061873909.  @jlebon, are there other paths we should also support?  Note that we're currently failing, not warning, on objectionable paths.

Reply:
Won't this regex also fail on just `/var`? We have a test for this [in FCOS](https://github.com/coreos/fedora-coreos-config/blob/52b1d1adfc7668bba1cc28bc14bd03a35de2ced4/tests/kola/var-mount/simple/config.bu#L17), but CoreOS CI doesn't run here.

Not sure on the use cases for mounting something under `/etc/` has, but agree it should work.

I think that should cover it otherwise! As long as we're applying this to just mount points (which it looks like that's the case), https://github.com/coreos/butane/issues/240#issuecomment-1061873909 is not a concern.

Reply:
ill replace the regex with '/(etc|var)' that seems to work for both paths

Reply:
It also allows `/varnish`.  Also, since you've removed the anchor, it allows `/foo/var`.  How about `^/(etc|var)(/|$)` ?